template/achievementbadge.html

```diff
@@ -0,0 +1,19 @@
+<link rel="import" href="data/achievementinfo.html">
+
+<dom-module id="papan-achievement-badge">
+  <link rel="import" type="css" href="achievementbadge.css">
+  <template>
+    <achievement-info achievementid={{achievementid}} achievementinfo={{achievement}}></achievement-info>
+
+    <div class="user-achievement">
+      <div class="pic">[[achievement.achievementpicture]]</div>[[achievement.achievementname]]
+    </div>
+  </template>
+
+  <script>
+    Polymer({
+      is: "papan-achievement-badge",
```

Comment:
Copy/paste stuff, the original is probably profile, in which I removed so many stuff that this line unexpectedly became final.